src/main/java/seedu/easylog/commands/itemscommands/ItemsAddCommand.java

```diff
@@ -28,28 +24,26 @@
      * @param itemDescriptionRecord ArrayList of item descriptions
      */
     public void execute(String itemDescription, ItemManager itemManager, ArrayList<String> itemDescriptionRecord)
-            throws EmptyNameException, EmptyItemPriceException, InvalidItemPriceException, NullItemPriceException,
-            NullItemStockException, EmptyItemStockException, InvalidItemStockException, InvalidTotalItemStockException,
-            NonNumericItemPriceException, NonIntegerNumericItemStockException {
-        if (itemDescription.equals("")) {
-            throw new EmptyNameException();
+            throws EmptyItemNameException, NonIntegerItemStockException, NonNumericItemPriceException,
+            InvalidTotalItemStockException, NullItemNameException, NullItemStockException, EmptyItemStockException,
+            InvalidItemStockException {
+        if (itemDescription == null) {
+            throw new NullItemNameException();
+        }
+
+        String itemDescriptionAfterTrimming = itemDescription.trim();
```

Comment:
no need for item description to be trimmed because inputs are alrd trimmed in the Ui askForUserInput() method.

Reply:
I see, will make changes to it :)